packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -414,32 +419,37 @@ export class AtomicSearchBox {
     );
   }
 
-  private isPanelInFocus(panel: 'right' | 'left') {
+  private isPanelInFocus(
+    panel: HTMLElement | undefined,
+    query?: string
+  ): boolean {
     if (!this.activeDescendantElement) {
       return false;
     }
-    return this.activeDescendantElement
-      ?.closest('ul')
-      ?.getAttribute('part')
-      ?.includes(`suggestions-${panel}`);
+
+    if (query) {
+      return !!panel?.querySelector(`li[data-query="${query}"]`);
```

Comment:
you should escape double-quotes in `query`.

Reply:
we should do that here too, then?
```jsx
        {...(hasQuery && {
          role: 'option',
          'data-query': item.query,
          'aria-selected': `${isSelected}`,
        })}
```

Reply:
no, specifically in `querySelectors` since the selector uses a quotation mark.

Reply:
ah yeah, duh

Reply:
this is the best I could do for you at this time 😂 https://github.com/coveo/ui-kit/pull/2060/commits/536c3ef4b81bb1f81203ce395205cadb3bcf6869 if you have any other suggestions, let me know!